models/unit.go

```diff
@@ -20,6 +20,40 @@ const (
 	UnitTypeExternalTracker                     // 9 ExternalTracker
 )
 
+var (
+	// allRepUnitTypes contains all the unit types
+	allRepUnitTypes = []UnitType{
+		UnitTypeCode,
+		UnitTypeIssues,
+		UnitTypePullRequests,
+		UnitTypeCommits,
+		UnitTypeReleases,
+		UnitTypeWiki,
+		UnitTypeSettings,
+		UnitTypeExternalWiki,
+		UnitTypeExternalTracker,
```

Comment:
Do we really need External as a separate unit? Can we enable both internal wiki and external wiki at the same time? (same for Tracker)